sway-lsp/src/server.rs

```diff
@@ -186,10 +178,8 @@ impl Backend {
 #[tower_lsp::async_trait]
 impl LanguageServer for Backend {
     async fn initialize(&self, params: InitializeParams) -> jsonrpc::Result<InitializeResult> {
-        eprintln!("InitializeParams = {:#?}", params);
-        self.client
-            .log_message(MessageType::INFO, "Initializing the Sway Language Server")
-            .await;
+        tracing::error!("InitializeParams = {:#?}", params);
```

Comment:
The existing `eprintln!("InitializeParams = {:#?}", params);` was accidentally left over from a previous PR. Can we remove this line?